README.md

```diff
@@ -1 +1,50 @@
-# google-cloud-exec-ruby
+# Google Serverless Execution Tool
+
+This repository contains the "google-serverless-exec" gem, a collection of libraries and plugins for integrating Ruby apps with Google serverless environments. It is not required for deploying a Ruby application to Google serverless compute, but it provides a number of convenience hooks and tools for integrating into Google serverless environments.
```

Comment:
The old appengine gem was intended to be a "collection" of plugins and libraries, but serverless-exec just does remote exec functionality, for example to perform database migrations in a production serverless environment.